src/System.Diagnostics.Debug/tests/Configurations.props

```diff
@@ -4,6 +4,9 @@
     <BuildConfigurations>
       netcoreapp-Unix;
       netcoreapp-Windows_NT;
+      uap-Windows_NT;
```

Comment:
I don't really see the point of adding uap, uapaot and netfx configurations, when all of the builds are the same. Can we instead have only a netstandard configuration? Also, I'm not sure but we would have to check if this tests pass in uapaot. Unfortunatelly you can't run those tests yet yourself, but I'll pull your branch and try it out.